.env

```diff
@@ -51,7 +51,7 @@ HASH_TASK_EXECUTOR_PORT=5010
 LOG_LEVEL=info
 
 SYSTEM_USER_SHORTNAME=system-user
-SYSTEM_USER_PREFERRED_NAME=System User
+SYSTEM_USER_PREFERRED_NAME="System User"
```

Comment:
This seems problematic, why did we have to add quotes?

Reply:
Without quotes, this is not a valid `.env` file. Many parsers may allow it, but the Rust crate for parsing `.env` files does not.

Reply:
I see, what sort of error message appears without it? Will CI pick it up?

Just trying to think about the cognitive overhead of how users define these, also this PR might break quite a few of people's current `.env.local` setups so it'd be good if the error message told them, otherwise we need to announce it 😅

Reply:
CI will fail as `just` won't be able to parse it.
```
❯ just
error: Failed to load environment file: Error parsing line: 'System User', error at line index: 7
```

Reply:
Hm, that's probably fine, I do wish it would say something about spaces.

Thanks!